roottest/root/io/newstl/ComplexTest_win64.ref

```diff
@@ -0,0 +1,8 @@
+
+Processing runComplexTest.C...
+Warning in <TStreamerInfo::Build>: _Complex_base<double,_C_double_complex>: base class _C_double_complex has no streamer or dictionary it will not be saved
+Warning in <TStreamerInfo::Build>: complex<double>: base class _Complex_base<double,_C_double_complex> has no streamer or dictionary it will not be saved
```

Comment:
Those are an issue per se.  I realize that the test was not run on Windows before and so we did not noticed.  We should open an issue to track the needed removal of those (i.e. by updating the I/O code to properly handle std::complex that have the same implementation as Windows).